src/test/scala/is/hail/methods/ExprSuite.scala

```diff
@@ -789,13 +789,13 @@ class ExprSuite extends SparkSuite {
       x.get(false) should contain theSameElementsAs Seq(1, 3, 5)
     }
 
-    (eval[Map[String, IndexedSeq[Int]]]("[1].filter(k => false).groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")").get
+    (eval[Map[String, IndexedSeq[Int]]]("[].groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")").get
```

Comment:
Oh, this is the cause of the test error. My assumption was right that the old expression language doesn't allow "[]". I thought I had checked this in IntelliJ, but it didn't actually run any tests (I seem to only be able to run entire test classes, not individual test methods).

This used to be "[1].tail()", which evaluates to the empty array, as a way around not being able to write "[]" directly. Since I removed tail, I needed another way to construct an empty array.